packages/itwinui-react/src/core/utils/functions/import.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Wrapper around native `import()` with workarounds for:
+ *   1. typescript trying to transpile dynamic import in CJS environment (https://github.com/microsoft/TypeScript/issues/43329)
+ *   2. jest/node causing segfault when encountering dynamic import (https://github.com/nodejs/node/issues/35889)
+ */
+export const dynamicImport =
```

Comment:
its not really dynamic import tho is it because youre not awaiting it right?

Reply:
it is just a wrapper over `import()`, but its main use case is dynamic imports.

the `await`ing happens in the place where it is called.

Reply:
yea exactly so the name is misleading, just a nit

Reply:
Is it misleading though? For who? 😄 

From MDN:

> The import() syntax, commonly called dynamic import [...]

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import

Reply:
wow; TIL!